app/interactors/login_user/generate_jwt.rb

```diff
@@ -0,0 +1,21 @@
+class LoginUser
+  class GenerateJwt
+    include Interactor
+
+    delegate :user, to: :context
+
+    def call
+      context.token = JwtCodec.new.encode(user_attributes)
```

Comment:
This information is usually called `payload`. But you can leave your attribute name. It doesn't matter much.

```suggestion
      context.token = JwtCodec.new.encode(payload)
```